scikeras/wrappers.py

```diff
@@ -834,6 +855,9 @@ def postprocess_y(self, y):
 
         y = np.squeeze(np.column_stack(class_predictions))
 
+        # type cast back to input dtype
+        y = y.astype(self.input_dtype_, copy=False)
```

Comment:
:+1: This will preserve the input/output dtype of `y`, which might be important for pipelines.

I've added the flag `copy=False`. There's no need to copy if it's of the same dtype (which is what `astype` does by default).

Reply:
Good catch. Small optimizations add up!